### ISSUE TITLE
Use getVersionName instead getVersionCode for apk exports

### ISSUE THREAD

**Issue by balporig** [SmartPack/PackageManager#82]
The implementation of #76 uses `sAPKUtils.getVersionCode` to prevent overwrites which for that purpose works fine, but as in most time it does not correspond to the actual app version it's not very useful.  My suggestion is to use the human-readable ` sAPKUtils.getVersionName` instead, just as for the `Export Details` and `App Info`. Here's an example app `Elevate` with version `5.60.1`:

| app                    | version        |
| ---------------------- | -------------- |
| com.wonder.apkm        | no version     |
| com.wonder_2450.apkm   | getVersionCode |
| com.wonder_5.60.1.apkm | getVersionName |

**Comment by sunilpaulmathew** [comment]
@balporig 
I understand & I'll think about it. Actually, version code (not version name) is the only thing supposed to change on every app update.

**Comment by balporig** [comment]
Yep, you're right, but usually only the version name is exposed to end users

**Comment by Pantyhose-X** [comment]
Should have used #95 Solution
